src/catkin_pkg/package.py

```diff
@@ -253,6 +253,9 @@ def validate(self, warnings=None):
 
         if not self.description:
             errors.append('Package description must not be empty')
+        else:
+            if '\n' in self.description:
+                new_warnings.append('Package "%s" has newlines in the description')
```

Comment:
I'm not sure we want to do this. Just because setuptools doesn't support newlines in descriptions doesn't mean we should discourage package manifests from containing them. A quick survey shows that this warning will affect ~15% of packages in ROS 2.

I think it would be better to silently convert the newlines to spaces.

Reply:
Additionally, the package name is missing from this format string.

Reply:
+1 on silent conversion. I would even be tempted to version-scope that so it only happens on setuptools >= 59.1 so that packages on current platforms remain unaffected.